### PR TITLE
Subscribe token price

### PR DESCRIPTION
**Pull request by drondin**
Allows users to subscribe on multiple token price feeds

New Buttons:
image

The "selected" token is stored in the user profile. There is no easy way to ask multiple questions without storing data in the profile.

The tokens that the user is subscribed to is in a different table, if they want to unsubscribe they need to put a high value like 1000.

### CODE REVIEW

**Comment by drondin** [comment]
Don't merge yet, I have found a bug

**Comment by drondin** [comment]
The PR is ready to review

File: src/main/java/pro/belbix/tgnotifier/tg/Commands.java [hunk 1]
```diff
@@ -152,12 +168,44 @@ public static boolean fillFieldForCommand(@NotNull String command, @NotNull User
...
             return false;
         }
         return true;
     }
 
+    private static void insertOrUpdateToken(UserEntity userEntity, String tokenName, Double change){
+        if (tokenName==null){
+            throw new IllegalStateException("Token not selected");
+        }
+        else{
```
Comment (belbix) [inline src/main/java/pro/belbix/tgnotifier/tg/Commands.java:185]: else claus not necessary

File: src/main/java/pro/belbix/tgnotifier/tg/Commands.java [hunk 2]
```diff
@@ -152,12 +168,44 @@ public static boolean fillFieldForCommand(@NotNull String command, @NotNull User
...
+
+            if (!found){
+                TokenWatchEntity newToken = new TokenWatchEntity();
+                newToken.setTokenName(tokenName);
+                newToken.setPriceChange(change);
+                newToken.setUser(userEntity);
+                userTokens.add(newToken);
+            }
+            
+            userEntity.setTokenWatchList(userTokens);
```
Comment (belbix) [inline src/main/java/pro/belbix/tgnotifier/tg/Commands.java:205]: userTokens the same, remove

File: src/main/java/pro/belbix/tgnotifier/tg/Commands.java [hunk 3]
```diff
@@ -152,12 +168,44 @@ public static boolean fillFieldForCommand(@NotNull String command, @NotNull User
...
         }
         return true;
     }
 
+    private static void insertOrUpdateToken(UserEntity userEntity, String tokenName, Double change){
+        if (tokenName==null){
+            throw new IllegalStateException("Token not selected");
+        }
+        else{
+            List<TokenWatchEntity> userTokens = userEntity.getTokenWatchList();
```
Comment (belbix) [inline src/main/java/pro/belbix/tgnotifier/tg/Commands.java:186]: you can simplify it with stream

**Comment by drondin** [comment]
I think it might be ready to merge now, let me know what you think

**Pull request merged**